src/assets/scss/_mixins.scss

```diff
@@ -128,3 +128,14 @@
 
   & > * { pointer-events: auto; }
 }
+
+@mixin overlay($position, $opacity) {
```

Comment:
How about using this mixin in `FormAttachmentPopups` as well? Just to make it clear what the related usage sites are. Could be a follow-up PR as well.